Moab.Models/Helpers/ExerciseImportHelper.cs

```diff
@@ -20,17 +22,159 @@ public class ExerciseImportHelper
         {
+            string checkHeader = "ExerciseCode,Name,CDT_Class,CDT_AtHome,IsMovementDataCollected,UnitTarget,HintEasier,HintHarder,Hint1,Hint2,MDT_Class,MDT_AtHome,OldCode,Name_animationFile,Old_Name_animationFile";
+            if (header == checkHeader)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        protected Exercise FindExtantExerciseInCollection(ICollection<Exercise> exercises, string exerciseCode)
+        {
+           foreach (Exercise exercise in exercises)
+            {
+                if (exercise.ExerciseCode == exerciseCode)
+                {
+                    return exercise;
+                }
+            }
+            return null;
+        }
+
+        protected void UpdateExercise(Exercise exercise, string[] updateCSV)
+        {
+            //TODO: add support for  generic hint collection
+            exercise.ExerciseCode = updateCSV[(int)ExerciseCSVColumns.ExerciseCode];
+            exercise.Name = updateCSV[(int)ExerciseCSVColumns.Name];
+            exercise.EasierHint = updateCSV[(int)ExerciseCSVColumns.HintEasier];
+            exercise.HarderHint = updateCSV[(int)ExerciseCSVColumns.HintHarder];
+            exercise.HasRepetitionTarget = ConvertYNtoBool(updateCSV[(int)ExerciseCSVColumns.UnitTarget]);
+            exercise.DateLastUpdated = DateTime.Now;
+        }
+
+        // Converts Y into true and N into false
+        protected bool ConvertYNtoBool(string input)
+        {
+            if (input.ToUpper() == "Y")
+            {
+                return true;
+            }
+            else if (input.ToUpper() == "N")
+            {
+                return false;
+            }
             return false;
         }
 
+        // Converts the input string into a
+        protected List<string[]> SplitCSVInput(string CSVInput)
+        {
+            // Create LineList
+            List<string[]> LineList = new List<string[]>();
+            int iterator = 0;
+            do
+            {
+                string[] line = new string[14];
+                int iteratorNext = CSVInput.IndexOf('\n', iterator);
+                string temp;
+                if (iteratorNext == -1)
+                {
+
+                    temp = CSVInput.Substring(iterator, CSVInput.Length - iterator);
+                }
+                else
+                {
+                    temp = CSVInput.Substring(iterator, iteratorNext - iterator);
+                }
+                line = SplitCSVLine(temp);
+                LineList.Add(line);
+                iterator = ++iteratorNext;
+            }
+            while (iterator > 0);
+            if (LineList.Count == 0)
+            {
+                throw new FormatException("Invalid Format of CSV Input");
+            }
+
+            // Delete Header and Empty Row
+            LineList.RemoveRange(0, 2);
```

Comment:
We've got a logic problem here.  You remove the header line before it has been checked.
I think the input file has an extra line between the header and the data by mistake.  I think you should update this method to remove all blank lines.